### ISSUE TITLE
[Request] Example data

### ISSUE THREAD

**Issue by austinmw** [tensorflow/tcav#42]
Would it be possible to provide links to an example of required model files and concept/target data? It would be much easier for people to test out with a self-contained example.

**Comment by justcho5** [comment]
Yes, I was looking for this as well... I'm unsure of the images that are needed for the source directory. The notebook says the following:

> source_dir: where images of concepts, target class and random images (negative samples when learning CAVs) live. Each should be a sub-folder within this directory.
> 
> Note that random image directories can be in any name. In this example, we are using random500_0, random500_1,.. for an arbitrary reason.
> 
> You need roughly 50-200 images per concept and target class (10-20 pictures also tend to work, but 200 is pretty safe).

For an example such as zebra:
Concept directory: folder with images of stripes
Target class directory: folder with images of zebra
random images directory: Are these images random images sampled from whole dataset, without considering their labels? Can they end up having zebra images?

Thank you!

**Comment by jameswex** [comment]
With respect to image sizes, that depends on the model and activation generator you are providing to TCAV and if that can take images of different sizes and get activations out of the model. If you look at the provided ImageActivationGenerator code (https://github.com/tensorflow/tcav/blob/master/tcav/activation_generator.py#L95), it resizes any loaded images to be the correct size for the model it is running TCAV on.

@BeenKim can you comment on the random images and stripe images you used in the initial research and whether those images came from training data or another dataset? And whether the striped images you used were simple stripe patterns or contained objects (such as zebras) that themselves were striped?

**Comment by justcho5** [comment]
Great! Thank you for the response. As a test, I used random images from Google for the random folders. This seemed to work with the tutorial notebook. However, I would still like to know if the random images can and should be from the whole dataset (which would also include the target class images).

**Comment by BeenKim** [comment]
Hi all!
All my images are sampled from http://www.image-net.org/. I did a simple wget via their URLS http://image-net.org/download-imageurls, then filter based on the size of the file to remove obsolete link (some images they used have on flickr is no longer there). 

I also got zebra images from  http://www.image-net.org/ zebra class. For striped images, I got them from the broaden dataset, that you can download from here: https://github.com/CSAILVision/NetDissect/tree/release1/dataset

Some practical problem that why we could not repost our examples here is the IP issue - we (google) are not allowed to re-distribute imagenet images. :( 
Hope this helps!
Been
